spec/features/user_searches_voters_spec.rb

```diff
@@ -34,10 +34,20 @@
 
     click_button 'Search'
 
-    expect(page).to have_content('10 voters found')
-    @voters.each do |voter|
+    expect(page).to have_content('51 voters found')
+    @voters[0..24].each do |voter|
       expect(page).to have_content(voter.first_name)
     end
+
+    click_link 'Next'
+
+    @voters[25..49].each do |voter|
+      expect(page).to have_content(voter.first_name)
+    end
+
+    click_link 'Next'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.